cetmix_tower_server/tests/test_command.py

```diff
@@ -41,6 +41,19 @@ def setUp(self, *args, **kwargs):
     COMMAND_RESULT = {"exit_code": 0, "message": "New command was created"}
 else:
     COMMAND_RESULT = {"exit_code": -1, "message": "error"}
+    """,
+            }
+        )
+
+        self.command_create_new_command_2 = self.Command.create(
+            {
+                "name": "Create new command",
```

Comment:
Command name as well as the name of the variable it's assigned don't look relevant. Which degraded the code readability.
There should be something like "Python command with secret" or similar.

Reply:
Thanks, I'll fix it